src/errors/process-test-fn-error.js

```diff
@@ -8,6 +10,13 @@ import {
 } from './test-run';
 
 
+function isAssertionErrorCallsiteFrame (frame) {
+    var filename = frame.getFileName();
+
+    return !new RegExp(`${escapeRe(sep)}node_modules${escapeRe(sep)}`).test(filename);
```

Comment:
We need to filter node internal modules as well see: https://github.com/DevExpress/testcafe/blob/master/src/errors/stack-filter.js#L23 and https://github.com/DevExpress/testcafe/blob/master/src/errors/stack-filter.js#L24